joomla/joomla-cms
# com_wrapper XML coding standard

#### Description

This PR makes cosmetic changes to **com_wrapper** XML files to be in compliance with XML coding standards (Check https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md).

Only an extra change has been made: Added missing `description="COM_WRAPPER_FIELD_FRAME_DESC"`
#### How to test

This is a cosmic change. A simple view of the changes should be enough.
But for test you can check if the wrapper component work as before.
#### Observations

Any improvements are welcome
